consoles/prometheus.html

```diff
@@ -22,6 +23,7 @@
 <tr>
   <td><a href="prometheus-overview.html?instance={{ .Labels.instance }}">{{ .Labels.instance }}</a></td>
   <td {{ if eq (. | value) 1.0 }}>Yes{{ else }} class="alert-danger">No{{ end }}</td>
+  <td {{ if eq (printf "prometheus_local_storage_rushed_mode{job='prometheus',instance='%s'}" .Labels.instance | query | first | value) 1.0 }} class="alert-danger">Yes{{ else }}>No{{ end }}</td>
```

Comment:
By the time it's 1.0 you're already fallen over, this would be have too many false negatives.